webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -92,6 +94,23 @@ export default class Logs extends React.PureComponent<Props, State> {
         this.setState({page: this.state.page - 1});
     };
 
+    downloadLogs = async () => {
+        let logString;
+        if (this.props.isPlainLogs) {
+            await this.props.actions.getAllPlainLogs();
+            logString = this.props.allPlainLogs.join('');
```

Comment:
Assuming we don't move this to the server though, instead of storing all of the plain logs in the Redux store and passing them in as a prop, could we actually change this to something more like this where we just get the response directly from the action? I don't think we need to save all the logs in the Redux state since this is a one-off action, so we don't need to store them in Redux state
```js
            const response = await this.props.actions.getAllPlainLogs();
            if (!response.data) {
                // handle error somehow
            }
            logString = response.data.join('');
```

I'm a bit surprised that this worked as-is. I know we await on the action, but I would've expected `this.props.allPlainLogs` to have been the logs from when `downloadLogs` is first called which would be before we actually loaded them